.github/workflows/canary-integration-test.yml

```diff
@@ -1512,6 +1512,9 @@ jobs:
       - name: setup cluster resources
         uses: ./.github/workflows/canary-test-config
 
+      - name: allow holder pod deployment
+        run: sed -i "s|CSI_DISABLE_HOLDER_PODS|# CSI_DISABLE_HOLDER_PODS|g" "deploy/examples/operator.yaml"
+
```

Comment:
For now, let's allow holder pods to get the CI working. This will make sure legacy behavior is preserved. Let's follow up in a new PR to ensure the new behavior works greenfield beyond the manual testing I've done.